pwndbg/commands/next.py

```diff
@@ -36,6 +36,13 @@ def nextcall(*args):
 
 @pwndbg.commands.Command
 @pwndbg.commands.OnlyWhenRunning
+def nextsymbol(*args):
```

Comment:
Instead of defining a new command, modify the existing `nextcall` command.

Thanks!